src/components/AccessibilityGuide/StickyElements.mdx

```diff
@@ -0,0 +1,105 @@
+import { Meta, Source } from "@storybook/blocks";
+
+import Link from "../Link/Link";
+
+<Meta title="Accessibility Guide/Sticky Elements" />
+
+# Sticky Elements
+
+## Table of Contents
+
+- {<Link href="#general-information" target="_self">General Information</Link>}
+- {<Link href="#general-issues" target="_self">General Issues</Link>}
+- {<Link href="#implementation" target="_self">Implementation</Link>}
+- {<Link href="#resources" target="_self">Resources</Link>}
+
+## General Information
+
+Sticky elements switch between relative and fixed position. They move with the viewport as the user scrolls, so they are always visible (until they hit a given breakpoint).
+
+Generally, we want to avoid using sticky elements. They should be used only for critical information that needs to remain visible, or to allow a continuous process for the user.
+The most common use case, a sticky header, is only useful when users will be repeatedly accessing the menu and scrolling quickly through multiple pages. This is why you often see
+them used on e-commerce sites. If users don't need immediate navigation and will generally stay on one or two pages, a sticky header is unnecessary. Additionally, sticky headers can
+obscure other elements on the page. If you need a large menu that will inevitably cover parts of the page, consider an accordion or just making it fixed.
```

Comment:
I see now that you are making a distinction between `position: sticky` and `position: fixed`. Can you include a brief explanation telling why using `fixed` would be better for this situation?